core/src/Kokkos_OpenMP.hpp

```diff
@@ -135,8 +137,7 @@ class OpenMP {
       int requested_partition_size = 0);
 #endif
 
-  // use UniqueToken
-  static int concurrency();
+  static int concurrency(OpenMP const& = OpenMP());
```

Comment:
Why can't this be a non-static member function as it kind of is supposed to be?

Reply:
I did that for consistency. The function exists in several backends and it is always static.

Reply:
OK we need an immediate follow up on this: this is ONLY static in the other guys because someone suggested that it is the same for every instance of currently existing types: it should NOT be static.